Better standalone error message?

When using the standalone, I get:

```
In file included from ./include/boost/multiprecision/mpfr.hpp:9:
In file included from ./include/boost/multiprecision/number.hpp:10:
./include/boost/multiprecision/detail/standalone_config.hpp:15:10: fatal error: 'boost/config.hpp' file not found
#include <boost/config.hpp>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
```

This *is* an expected error:

```
// Boost.Config is dependency free so it is considered a requirement to use Boost.Multiprecision in standalone mode
#include <boost/config.hpp>
```

but could it give a nice error message as to why it's not supported using `__has_include`?

--- a/include/boost/multiprecision/detail/standalone_config.hpp
+++ b/include/boost/multiprecision/detail/standalone_config.hpp
@@ -11,9 +11,20 @@
 #define BOOST_MP_STANDALONE_CONFIG_HPP
 
 #include <climits>
+
 // Boost.Config is dependency free so it is considered a requirement to use Boost.Multiprecision in standalone mode
-#include <boost/config.hpp>
-#include <boost/config/workaround.hpp>
+#ifdef __has_include
+#  if __has_include(<boost/config.hpp>)
+#    include <boost/config.hpp>
+#    include <boost/config/workaround.hpp>
+#  else
+#    error "Boost.Config is considered a requirement to use Boost.Multiprecision in standalone mode. A package is provided at https://github.com/boostorg/multiprecision/releases"
+#  endif
+#else
+// Provides the less helpful fatal error: 'boost/config.hpp' file not found if not available
+#  include <boost/config.hpp>
+#  include <boost/config/workaround.hpp>
+#endif
 
 // Minimum language standard transition
  #ifdef _MSVC_LANG

